API.md

```diff
@@ -137,10 +137,16 @@ The following options are supported:
 By default, the handler will look for a page template called `layout.html` on your configured Vision path,
 but if you use a template in a different location or with a different name you can use this option:
 
-  - `pageTemplateName` - a string specifying the filename of an alternative page template to use, rather than the
+  - `pageTemplateName` - a string specifying the filename of an alternative layout to use, rather than the
     default `layout.html`.
 
-Note that this option is ignored on any routes where you specify an entirely bespoke view using the `viewName` option. 
```

Comment:
This didn't tally with my experience when using the plugin - pageTemplateName only changes the surrounding layout for the page, setting a custom `viewName` still used the default layout. 
This statement is true for the added config option, so it's been reworded slightly in the section below.